roles/nighthawk/templates/http-service.yml.j2

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: nighthawk-benchmark
+  name: nighthawk-benchmark-service-{{ type }}-{{ item }}
+  namespace: "{{ operator_namespace }}"
+spec:
+  ports:
+  - name: http
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: nginx-{{ type }}-{{ item }}
+  type: {{ workload_args.service_type | default("ClusterIP") }}
```

Comment:
You can remove all references to NodePort and just left the default, ClusterIP, ingress traffic flow is not affected by the type of service backing the route(s)

Reply:
Keeping ClusterIP as default as per the suggestion. Thanks